### PR TITLE
Animated emojis, armor crash fix, json apis

### PR DESCRIPTION
**Pull request by omoflop**
title

### CODE REVIEW

**Comment by Covkie** [comment]
If anyone has no objections ill merge this

**Comment by UnlikePaladin** [comment]
Why is this removing the GoL easter egg without mentioning it anywhere?

**Comment by UnlikePaladin** [comment]
If anything debug mode should be a regular config option on the settings screen and not be hidden behind a key combo that is famously used for easter eggs

File: fabric/src/main/java/org/figuramc/figura/config/fabric/ModMenuConfig.java [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 import com.terraformersmc.modmenu.api.ModMenuApi;
 import org.figuramc.figura.FiguraMod;
+import org.figuramc.figura.config.Configs;
 import org.figuramc.figura.gui.screens.ConfigScreen;
 
 public class ModMenuConfig implements ModMenuApi {
     @Override
     public ConfigScreenFactory<?> getModConfigScreenFactory() {
-        return parentScreen -> new ConfigScreen(parentScreen, FiguraMod.DEBUG_MODE);
+        return parentScreen -> new ConfigScreen(parentScreen, Configs.DEBUG_MODE.value);
```
Comment (UnlikePaladin) [inline fabric/src/main/java/org/figuramc/figura/config/fabric/ModMenuConfig.java]: You forgot to change this to your FiguraMod.debugModeEnabled()

File: common/src/main/java/org/figuramc/figura/gui/screens/AbstractPanelScreen.java [hunk 2]
```diff
@@ -216,7 +216,8 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
         egg += (char) keyCode;
         egg = egg.substring(1);
         if (EGG.equals(egg)) {
-            Minecraft.getInstance().setScreen(new GameScreen(this));
+            Configs.DEBUG_MODE.setValue("true");
```
Comment (UnlikePaladin) [inline common/src/main/java/org/figuramc/figura/gui/screens/AbstractPanelScreen.java]: Might as well make it a regular option on the screen, it doesn't need to be hidden behind a key combo. We already expose several options normal users don't mess with like the ip backend option.

File: common/src/main/java/org/figuramc/figura/FiguraMod.java [hunk 3]
```diff
@@ -49,6 +48,13 @@ public class FiguraMod {
     public static boolean parseMessages = true;
     public static boolean processingKeybind;
 
+    public static boolean debugModeEnabled() {
+        if (Configs.DEBUG_MODE == null) return false;
+        Boolean b = Configs.DEBUG_MODE.value;
+        if (b == null) return false;
+        return b;
```
Comment (UnlikePaladin) [inline common/src/main/java/org/figuramc/figura/FiguraMod.java]: you do know you could simplify this entire method to
```suggestion
   return b == true;
```
lol

**Review by UnlikePaladin — DISMISSED** [review]
The rest of the changes relating to animated emojis look good to me, just these minor things.

**Comment by omoflop** [comment]
@UnlikePaladin thanks for the feedback, will fix all that stuff 👍

**Pull request merged**